### DISCUSSION TITLE
Get all keys of keyboard node in script canvas?!

### DISCUSSION THREAD

**Discussion by watchmax3** [o3de/o3de#8147] [General]
Like when player type something in game...

**Comment by FiniteStateGit** [comment]
Create a UI canvas with a text input component:
![ui-editor-text-input](https://user-images.githubusercontent.com/31811688/158918698-249c2861-dc06-49fc-af65-f403112c91f6.png)
Note: I have the UI Slice Library gem installed, you don't need it though.

When the canvas is loaded in-game, you can type text into the text input field.  You don't need an input component with keyboard mappings.  Here are the Script Canvas nodes to work with the UI Text Input component:
![ui-text-input-nodes](https://user-images.githubusercontent.com/31811688/158918547-d9017244-e486-4e96-9676-e2f62229d255.png)

**Reply by watchmax3** [reply]
Thanks for answer and help.
